components/display/ili9341.rst

```diff
@@ -43,10 +43,10 @@ too (with some SPI flash changes).
 Configuration variables
 ***********************
 
-- **led_pin** (:ref:`Pin Schema <config-pin_schema>`): The display's backlight pin.
-- **cs_pin** (:ref:`Pin Schema <config-pin_schema>`): The CS pin.
-- **dc_pin** (:ref:`Pin Schema <config-pin_schema>`): The DC pin.
-- **reset_pin** (:ref:`Pin Schema <config-pin_schema>`): The RESET pin.
+- **led_pin** (**Required**:ref:`Pin Schema <config-pin_schema>`): The display's backlight pin. It may be connected through a 3.3K resistor to +3V rail this way we can save one pin on ESP
```

Comment:
this is optional in code, optional / required must match code see https://github.com/esphome/esphome/blob/d20caa9d60e91b3044467f16eedfef9ab89d1299/esphome/components/ili9341/display.py#L32